requirements/edx/custom.txt

```diff
@@ -12,7 +12,7 @@
 -e git+https://github.com/edx/edx-notifications.git@0.6.0#egg=edx-notifications==0.6.0
 -e git+https://github.com/open-craft/problem-builder.git@d312e9a587e4376b9afadb982f5fc28e4a9061fc#egg=problem-builder
 -e git+https://github.com/OfficeDev/xblock-officemix.git@86238f5968a08db005717dbddc346808f1ed3716#egg=xblock-officemix
--e git+https://github.com/open-craft/xblock-chat.git@d3218cb6c549f953749a2a246952bf7557ed4739#egg=xblock-chat
+-e git+https://github.com/open-craft/xblock-chat.git@5b9dabe0b7c90f32a3ffee40a3a744e9ddc9e6be#egg=xblock-chat
```

Comment:
Please set this to the latest merge commit: 4e1ec8b4778377288577fdb4208460a1bbb0cceb